collector/service.go

```diff
@@ -21,6 +23,10 @@ var (
 		"collector.service.services-where",
 		"WQL 'where' clause to use in WMI metrics query. Limits the response to the services you specify and reduces the size of the response.",
 	).Default("").String()
+	wmiDisabled = kingpin.Flag(
```

Comment:
if there is no a particular reason i would found enable flags easy to read , something like `collectWMI` that is false by default

Reply:
I agree that collectWMI gives more context

Reply:
👍